entrypoint.sh

```diff
@@ -3,7 +3,22 @@
 git config --global --add safe.directory "/github/workspace/$1"
 
 cd $1
-curl $3 | jq --arg APP_NAME "$4" '{"data":map(select(.appName == $APP_NAME))}' > $2
+export DATA=$(curl $3 | jq --arg APP_NAME "$4" '{"data":map(select(.appName == $APP_NAME))}')
+
+if [[ $2 =~ .*\.json ]]; then
+    echo -e "$DATA" > $2
+elif [[ $2 =~ .*\.swift ]]; then
+    echo -e "// This Source Code Form is subject to the terms of the Mozilla Public\n// License, v. 2.0. If a copy of the MPL was not distributed with this\n// file, You can obtain one at http://mozilla.org/MPL/2.0/\n\nimport Foundation\n\npublic struct FirstRunExperiments {\n    public static let value = \"\"\"$DATA\"\"\"\n}" > $2
+elif [[ $2 =~ .*\.kt ]]; then
+    if [[ -z $6 ]]; then
+        echo "Package is not specified. Package is a required value for the kotlin language."
+        exit 1
+    fi
+    echo -e "// This Source Code Form is subject to the terms of the Mozilla Public\n// License, v. 2.0. If a copy of the MPL was not distributed with this\n// file, You can obtain one at http://mozilla.org/MPL/2.0/\n\npackage $6\n\nconst val firstRunExperiments = \"\"\"$DATA\"\"\"" > $2
+else
+    echo "Language of output file is not supported. Please choose between 'kotlin', 'swift', or 'json'"
+    exit 1
+fi
```

Comment:
I think this part is an optimization too far.

My understanding is that this will produce `kt` files containing large string literals, and that the Kotlin compiler struggles with [strings constants if they don't fit into 16 bits](https://youtrack.jetbrains.com/issue/KT-9552/IllegalArgumentException-UTF8-string-too-large-for-a-String-const-property-with-too-long-value).

Given that: that corresponds to around 20 concurrent first run experiments, and the optimization is so slight, I think I'd recommend not going down this path.

Reply:
Fair enough -- I'm going to comment it out and we can revisit it later if it's necessary